.github/CODEOWNERS

```diff
@@ -136,12 +157,56 @@ src/applications/mhv/secure-messaging @department-of-veterans-affairs/vfs-mhv-se
 src/applications/mhv/landing-page @department-of-veterans-affairs/vfs-mhv-integration @department-of-veterans-affairs/va-cto-health-products
 src/applications/avs @department-of-veterans-affairs/va-cto-health-products
 
-# VA.gov Product Teams - Forms
+# Dependents Benefits
 
-src/applications/simple-forms @department-of-veterans-affairs/platform-va-product-forms
+src/applications/static-pages/dependency-verification @department-of-veterans-affairs/benefits-ce-dependents
+src/applications/static-pages/view-modify-dependent @department-of-veterans-affairs/benefits-ce-dependents
+src/applications/static-pages/view-modify-dependents @department-of-veterans-affairs/benefits-ce-dependents
 
-# Other
+# Flagship Mobile Reviewers
+
+src/applications/static-pages/subscription-creators @department-of-veterans-affairs/flagship-mobile-reviewers
+src/applications/static-pages/widget-creators @department-of-veterans-affairs/flagship-mobile-reviewers
+
+# VA CTO Health Products
+
+src/applications/static-pages/covid-vaccine-updates-cta @department-of-veterans-affairs/va-cto-health-products
+
+# VAOS
 
 src/applications/vaos @department-of-veterans-affairs/vfs-vaos-fe
+src/applications/static-pages/health-care-manage-benefits/schedule-view-va-appointments-page  @department-of-veterans-affairs/vfs-vaos-fe @department-of-veterans-affairs/vsa-public-websites-frontend
+
+# Other
+
 src/applications/dhp-connected-devices @department-of-veterans-affairs/digital-health-platform
 src/applications/virtual-agent @department-of-veterans-affairs/orchid
+
+
+
+# TODO: who owns these?
```

Comment:
Do we know what will happen in terms of autoassigning PR reviewers if we have paths in here without assigned owners? (Aka: do we need to assign owners to each of these lines before merging this PR?)

Reply:
I think the global Platform assignment at the top (`* @department-of-veterans-affairs/va-platform-cop-frontend`) will pick those up. But I'd like a Platform person to confirm.

Reply:
I should add - if these don't get assigned to anyone before we merge, I'll remove them from this file and they should fall into the catch-all anyway.

Reply:
@rsmithadhoc confirmed any unassigned will fall back to the global Platform assignment at the top of the file (`@department-of-veterans-affairs/va-platform-cop-frontend`) [Slack thread](https://dsva.slack.com/archives/CBU0KDSB1/p1695417070193389?thread_ts=1695134841.707539&cid=CBU0KDSB1)